src/utils/DBL.ts

```diff
@@ -79,29 +95,22 @@ export default class DiscordBots {
       user.voteCooldown = `${Date.now() + 43200000}`;
       await user.save();
 
-      const functionToEval = async (id: string, embedToSend: MessageEmbed) => {
-        let hasSend = false;
-        const userInShard = await this.client.users.fetch(id).catch();
-
-        if (userInShard && !hasSend) {
-          hasSend = true;
-          try {
-            await userInShard.send(embedToSend);
-          } catch {
-            // console.log('[DBL] Cannot send message to user');
-          }
-        }
+      const sendMessageToUser = async (id: string, embedToSend: MessageEmbed) => {
+        const userInShard = await this.client.users.fetch(id);
+        await userInShard.send(embedToSend);
       };
-      // @ts-ignore
-      await this.client.shard.broadcastEval(functionToEval(vote.user, embed));
+
+      sendMessageToUser(vote.user, embed).catch();
```

Comment:
Usar o catch sem passar um callback evita o erro no console?

Outra coisinha, se colocar o catch aqui fora, ele vai dar catch na promise la de dentro? a de Send?

Reply:
> Usar o catch sem passar um callback evita o erro no console?

sim, vc muda o resultado dele pra `null`, quando da erro

> Outra coisinha, se colocar o catch aqui fora, ele vai dar catch na promise la de dentro? a de Send?

não entendi a pergunta

Reply:
Pera ai q to xom sono KSJAKAKA amanha explico